packages/melody-compiler/__tests__/__fixtures__/success/mount.template

```diff
@@ -3,5 +3,36 @@
 {% mount 'foo.twig' as 'bar' with {foo: 'bar'} %}
 
 {% mount Foo as 'bar' %}
-{% mount Foo as 'bar' with {foo: 'bar'} %}
+{% mount async as 'bar' with {foo: 'bar'} %}
 
+{% mount async 'foo.twig' as 'bar' with {foo: 'bar'} %}
+  Loading... {{ err }}
```

Comment:
why is `err` used within the placeholder?

Reply:
To ensure that within the placeholder it is associated with the right context (`_context.err`) while in the `catch` clause it should be bound to the generated functions parameter.

So: To verify that it compiles correctly.